### PR TITLE
MARCIN!

### PR DESCRIPTION
**Pull request by Ucio94**
Pull request

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -5,24 +5,28 @@
...
 
-<body>
+<body class="align-items-stretch">
     <div class="wrapper">
-        <div id="plansza"></div>
-        <div id="obrazek">
-            <img id="grafika" src="../img/s0.jpg" alt=""></img>
+        <div class="container text-center align-items-stretch">
+            <div id="password" class=""></div>
+            <div id="obrazek">
```
Comment (marcingol1) [inline src/index.html]: polish name - change to english :)

File: src/index.html [hunk 2]
```diff
@@ -5,24 +5,28 @@
...
+                <img id="grafika" src="../img/s0.jpg"></img>
```
Comment (marcingol1) [inline src/index.html]: polish name - change to english :)

File: src/index.html [hunk 3]
```diff
@@ -5,24 +5,28 @@
...
+            </div>
+            <div class="row justify-content-center">
+                <div id="alfabet" class="col-6"></div>
```
Comment (marcingol1) [inline src/index.html]: polish name - change to english :)

File: src/index.html [hunk 4]
```diff
@@ -5,24 +5,28 @@
...
+            </div>
+            <div style="clear: both;"></div>
```
Comment (marcingol1) [inline src/index.html]: why that's needed? :)

File: src/script.js [hunk 5]
```diff
@@ -1,7 +1,8 @@
 window.onload = start;
 // 
 
-let password = prompt("Enter password", ""); //password
+// let password = prompt("Enter password", ""); //password
+let password = 'elo';
```
Comment (marcingol1) [inline src/script.js]: restore commented code :)

File: src/script.js [hunk 6]
```diff
@@ -1,7 +1,8 @@
...
 password = password.toUpperCase() //password do duzych liter
 let dlugosc = password.length; //odczytanie dlugosci hasla
 let fails = 0; //zerowanie liczenia zlych klikniec
```
Comment (marcingol1) [inline src/script.js]: polish name - change to english :)

File: src/script.js [hunk 7]
```diff
@@ -14,7 +15,7 @@ let passwordBlank = password.split(' ').map((i) => {
 
 // wypisanie hasła
 function typePassword() {
-    document.getElementById("plansza").innerHTML = passwordBlank;
+    document.getElementById("password").innerHTML = passwordBlank;
 }
 // Tablica liter (zapytać marcina)(całość z neta)
 let alphabet = [...Array(26).keys()].map(i => String.fromCharCode(i + 65));
```
Comment (marcingol1) [inline src/script.js]: polish name - change to english :)

File: src/script.js [hunk 8]
```diff
@@ -27,6 +28,9 @@ function start() {
     for (i = 0; i <= 25; i++) {
         let element = "lit" + i;
         divBuilder = divBuilder + '<div onclick="sprawdz(' + i + ')" class="litera" id="' + element + '">' + alphabet[i] + '</div>';
+        // if ((i + 1) % 7 === 0) {
+        //     divBuilder = divBuilder + '<div class=""></div>'
+        // }
```
Comment (marcingol1) [inline src/script.js]: uncomment or delete

File: src/index.html (hunk 4 above)
Comment (Ucio94) [reply src/index.html]: Cause before added bootstrap I used float:left so the style cleared the float and I forgot to delete it :)

Comment (marcingol1) [reply src/index.html]: so delete it : )

Comment (Ucio94) [reply src/index.html]: Consider it done

**Pull request merged**